config/routes.rb

```diff
@@ -1,13 +1,13 @@
 Rails.application.routes.draw do
-  devise_for :admin_users, ActiveAdmin::Devise.config
-  ActiveAdmin.routes(self)
   get 'welcome/index'
 
   mount_devise_token_auth_for 'User', at: 'auth'
   get 'books', to: 'books#index'
   get 'books/shows' , to: 'books#shows'
 
-  get 'rent', to: 'rent#list'
+  get 'rents', to: 'rent#index'
```

Comment:
Remember my comment about REST and the `resources` helper that Rails provides! If you are creating a "standard" endpoint (create, update, delete, index, show), you should probably use `resources :models` instead of creating the endpoint with `get :rents`